docs/troubleshooting.rst

```diff
@@ -0,0 +1,18 @@
+.. _troubleshooting:
+
+Troubleshooting
+===============
+
+.. _troubleshooting_ocr_language_files_missing:
+
+Consumer warns ``OCR for XX failed``
+------------------------------------
+
+If you find the OCR accuracy to be too low, and/or the document consumer warns that ``OCR for
+XX failed, but we're going to stick with what we've got since FORGIVING_OCR is enabled``, then you
+might need to install the `Tesseract language files
+<http://packages.ubuntu.com/search?keywords=tesseract-ocr>`_ marching your documents languages.
+
+As an example, if your documents are written in Spanish you may need to run::
```

Comment:
We can't assume that our users are all using Debian or some derivitive thereof, so this should probably say something like:

> As an example, if your documents are written in Spanish and you're running a Debian-based system, you may need to run::